senza/traffic.py

```diff
@@ -395,3 +396,12 @@ def inform_sns(arns: list, message: str, region):
     sns = boto3.client('sns', region_name=region)
     for sns_topic in sns_topics:
         sns.publish(TopicArn=sns_topic, Subject="SenzaTrafficRedirect", Message=jsonizer.encode((message)))
+
+
+def resolve_to_ip_addresses(dns_name: str) -> set:
+    try:
+        answers = dns.resolver.query(dns_name, 'A')
+    except:
```

Comment:
E.g. maybe the 3rd party library could raise some IO or OS errors and I did not thought of that...